### ISSUE TITLE
[Bug] Error in rendering grafana dashboard from external app . Grafana app tag not rendering properly

### ISSUE THREAD

**Issue by hemsush** [grafana/grafana#9593]
Read before posting: 

- Questions should be posted to https://community.grafana.com. Please search there and here on GitHub for similar issues before creating a new issue. 
- Checkout FAQ: https://community.grafana.com/c/howto/faq
- Checkout How to troubleshoot metric query issues: https://community.grafana.com/t/how-to-troubleshoot-metric-query-issues/50

Please prefix your title with [Bug] or [Feature request]. 

Please include this information:
- What Grafana version are you using?
- What datasource are you using?
- What OS are you running grafana on?
- What did you do?
- What was the expected result?
- What happened instead?
- If related to metric query / data viz:
  - Include raw network request & response: get by opening Chrome Dev Tools (F12, Ctrl+Shift+I on windows, Cmd+Opt+I on Mac), go the network tab.

**Comment by hemsush** [comment]
I have enabled the auth proxy and configured nginx proxy server .I am using https. From my Java client when I hit the page of a dashboard, I was able to the page with error "cannot read property of ‘pageClass’ of undefined". I could see windows.grafanaBootdata is perfect and all other things are there in html ,but the contents of grafana-app tag is different than the page source. I am getting issue similar to https://github.com/grafana/grafana/issues/5735
I was able to view dashboard via browser .but when I try the same in Java HTTP connection , I am getting the above issue.
